### PR TITLE
Update docs on building inputs

### PR DESCRIPTION
**Pull request by VJalili**
This PR gathers all the docs related to building inputs in one section, simplifies the descriptions, and improves their formatting. 

As a side update, it also fixes the mermaid version to the same version as Docusarus to avoid version conflicts.

### CODE REVIEW

File: website/docs/advanced/build_inputs.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+```
+
+
+### Generating a reference panel
+
+This section only applies to the single-sample mode. 
+New reference panels can be generated from a single run of the 
+`GATKSVPipelineBatch` workflow. 
+If using a Cromwell server, we recommend copying the outputs to a p
+ermanent location by adding the following option to the workflow configuration file:
```
Comment (mwalker174) [inline website/docs/advanced/build_inputs.md:75]:
```suggestion
If using a Cromwell server, we recommend copying the outputs to a 
permanent location by adding the following option to the workflow configuration file:
```

Comment (mwalker174) [reply website/docs/advanced/build_inputs.md:75]: Link "workflow configuration" to the Cromwell docs so it's clear which file we're referring to: https://cromwell.readthedocs.io/en/latest/wf_options/Overview/

File: website/docs/advanced/build_inputs.md [hunk 2]
```diff
@@ -0,0 +1,137 @@
+---
+title: Default inputs
+description: Building default inputs
```
Comment (mwalker174) [inline website/docs/advanced/build_inputs.md:3]:
```suggestion
title: Building inputs
description: Building work input json files
```

**Review by mwalker174 — APPROVED** [review]
Thanks @VJalili, just a few minor suggestions here.

**Comment by VJalili** [comment]
Thanks, @mwalker174!

**Pull request merged**